packages/svelte/src/reactivity/url.js

```diff
@@ -150,6 +151,10 @@ export class ReactiveURL extends URL {
 	toJSON() {
 		return this.href;
 	}
+
+	[INSPECT_SYMBOL]() {
```

Comment:
We should probably add these to the prototype only in DEV to reduce bundle size.